runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -192,9 +191,9 @@ public Dimension render(Graphics2D graphics)
 		for (GroundItem item : groundItemList)
 		{
 			final LocalPoint groundPoint = LocalPoint.fromWorld(client, item.getLocation());
-
-			if (groundPoint == null || Math.abs(localLocation.getX() - groundPoint.getX()) > MAX_DISTANCE
-					|| Math.abs(localLocation.getY() - groundPoint.getY()) > MAX_DISTANCE
+			final int localDrawDistance = client.getScene().getDrawDistance() * Perspective.LOCAL_TILE_SIZE;
```

Comment:
Probably want to declare this outside the loop as it *should* be the same for all items.

Reply:
Thanks for the suggestion! I've added in a commit for that.